templates/zerver/integrations/mastodon.md

```diff
@@ -0,0 +1,39 @@
+Fetch public posts (sometimes called "toots") of individual accounts or
+federated hashtags from Mastodon in Zulip! While Zulip doesn't integrate
+directly with ActivityPub or the overall Fediverse, some ActivityPub servers,
+like [Mastodon](https://joinmastodon.org/), publish various RSS feeds which can
+be followed in Zulip using the [RSS][1] or [Zapier][2] integrations.
+
+!!! tip ""
+
+    Note that due to the various interactions and complexities of federation,
+    following a hashtag on one homeserver does not guarantee all posts in the
+    entire Fediverse will be included in the feed, only that all public posts
+    that have federated to the instance in question will be included. For a
+    visual guide to how Mastodon's Public Timeline works, [see this Reddit
+    comment with an included
+    flowchart](https://www.reddit.com/r/Mastodon/comments/8if9r6/adding_other_mastodon_instance_feeds_to_my/dyrjnq4/).
```

Comment:
Hm, is there a more official / professional-looking resource we might be able to link to?

Reply:
We could also probably drop the reference altogether.

Reply:
We could also probably drop the reference altogether.

Reply:
I might just drop the reference: I can't find anything in mastodon/mastodon or on joinmastodon.org that document federation at that altitude in a human format.